core/src/main/java/de/danoeh/antennapod/core/widget/WidgetUpdater.java

```diff
@@ -189,6 +204,14 @@ public static void updateWidget(Context context, WidgetState widgetState) {
         }
     }
 
+    private static PendingIntent createProgressStringIntent(Context context) {
+        Intent startingIntent = new Intent(context, MediaButtonReceiver.class);
+        startingIntent.setAction(MediaButtonReceiver.NOTIFY_PROGRESS_STRING_RECEIVER);
+        startingIntent.putExtra(MediaButtonReceiver.DUMMY_VALUE,"DUMMY_VALUE");
```

Comment:
Maybe you can use `MediaButtonReceiver.DUMMY_VALUE` in place of `"DUMMY_VALUE"`

I assume with the extra, it won't work?

Reply:
@tonytamsf Okay